addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -1261,7 +1265,9 @@ var ManualModel = StatementModel.extend({
                 self.reconcileModels = reconcileModels;
             });
 
-        return $.when(def_reconcileModel, def_account).then(function () {
+        var def_taxes = this.loadTaxes();
```

Comment:
All rpc are already copied/pasted so... I don't think it's really relevant to put the taxes rpc in a separated method. I suggest to simply copy paste the rpc:

```
var def_taxes = this._rpc({
                model: 'account.tax',
                method: 'search_read',
                fields: ['price_include', 'amount_type'],
            })
            .then(function (taxes) {
                _.each(taxes, function(tax){
                    self.taxes[tax.id] = {
                        price_include: tax.price_include,
                        amount_type: tax.amount_type,
                    }
                })
            });
```

Reply:
I agree with @kebeclibre copy paste is the source of hell! NEVER copy paste!!! A good example is that some change have been made in master where we change the fields that were fetched for the taxes, if we fix this issue by copy paste, during forward port, only the statement reconciliation will be impacted with the changes in master and not the manual reconciliation as it was copy paste.. Hence more work for our lovely support and bugfix team later on and we don't want that.

Reply:
This PR is clearly coming from a bad design at the first place. A complete override of the `load` method is also very crappy and it's not by adding some additional methods you will improve this fact.